functions/utils.js

```diff
@@ -13,7 +18,7 @@ import { Octokit } from '@octokit/core'
  * @param {string} repo
```

Comment:
@theory-in-progress please remove these parameters from the comment as we have also removed them from the function